src/protocol/libp2p/kademlia/mod.rs

```diff
@@ -1101,6 +1125,31 @@ impl Kademlia {
                         None => return Err(Error::EssentialTaskClosed),
                     }
                 },
+                action = self.store.next_action() => match action {
```

Comment:
One other super small thingie not for this PR, we can try to move away from writing code in `tokio::select` since that breaks our indentation (ie use a `fn hande_store_action(..)` ). We can come later and tidy-up all other places